docs/hopsworksai/azure/aks_acr_integration.md

```diff
@@ -0,0 +1,229 @@
+# Integration with Azure AKS and ACR
+
+This guide shows how to create a cluster in hopsworks.ai with integrated support for Azure Kubernetes Service (AKS) and Azure Container Registry (ACR). This enables Hopsworks to launch Python jobs, Jupyter servers, and ML model servings on top of AKS.
+
+Hopsworks AKS and ACR integration have four requirements:
+
+1. A virutal network with access to AKS pods and the AKS API servers 
+2. One Azure container registry configured in your account
+3. One AKS cluster
+4. Permissions to the ACR and AKS attached to a user-managed identity
+
+This guide on example setup with a private AKS cluster and public ACR.
+!!! note
+A public AKS cluster means the kubernetes API server is accessible outside the virtual network it is deployed in. Similarly, a public ACR is accessible through the internet.
+
+
+## User-managed identity
+
+!!! note
+The managed identity is attached to the virtual machines that runs inside your subscription. Hence, the permissions only applies to services that runs within your subscription. 
+
+AKS and ACR integration requires some additional permissions to be attached to the managed identity used by the hopsworks cluster. 
+
+To setup the managed identity, go to managed identities in the azure portal and then click add. Place the managed indentity in the same resource group as your are planning to create the hopsworks cluster in. In addition, choose a name for the managed identity, then review and create.
+
+Next go to the created managed identity in order to create role assignments. Go to azure role assignments and add the following roles with scope of the resource group you are planning to use:
+
+* AcrPull
+* AcrPush
+* Azure Kubernetes Service Contributor Role
+
+!!! warning
+You will also need to attach the Storage Blob Data Owner role to the managed identity, see [Creating and configuring a storage](getting_started.md#step-2-creating-and-configuring-a-storage)
+
+Once finished the role assignments should look similar to the picture below. Note that hopsworks-stage is used as the resource group throughout the example setup.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-permissions.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-permissions.png" alt="AKS permissions">
+    </a>
+    <figcaption>AKS permissions</figcaption>
+  </figure>
+</p>
+
+## Private AKS cluster and public ACR
+
+This guide will step through setting up a private AKS cluster and a public ACR. 
+
+### Step 1 Create an AKS cluster
+Go to kubernetes services in the azure portal and click add. Place the kubernetes cluster in the same resource group as the hopsworks cluster will reside in and choose a name for the kubernetes cluster.
```

Comment:
(will reside in) is not needed here.

Reply:
you could write
the same resource group that the Hopsworks cluster is located in, and choose a...

Reply:
Fixed